templates/terraform/schema_property.erb

```diff
@@ -135,8 +135,9 @@
 <% unless property.default_value.nil? -%>
     Default: <%= go_literal(property.default_value) -%>,
 <% end -%>
-<% unless property.conflicts_with.empty? -%>
-    ConflictsWith: []string{<%= property.conflicts_with.map{ |p| go_literal(p) }.join(', ') -%>},
+<% unless property.conflicted_properties().empty? && property.conflicts_with.empty? -%>
```

Comment:
We don't need `conflicts_with` anymore, right? Want to remove it from the list of things that can be set in terraform mm?

Reply:
We actually do! Terraform was referencing this thing called `name-prefix`. I'm not sure what that means, but it's certainly not a MM thing. I had to keep conflicts_with in TF because TF will lose the ability to reference this name-prefix otherwise.